app/controllers/variables_controller.rb

```diff
@@ -80,6 +80,10 @@ def destroy
     end
   end
 
+  def download_json
+    send_data @variable, type: 'application/json; charset=utf-8', disposition: "attachment; filename=#{@concept_id}.json"
```

Comment:
The data downloaded is not in JSON format, its a Ruby hash object, you think this can be fixed by just adding `to_json` to your `@variable` here.